composer.json

```diff
@@ -15,15 +15,15 @@
         "php": ">=7.1.0",
         "atlas/info": "~1.0",
         "atlas/mapper": "~1.0",
-        "psr/log": "~1.0"
+        "psr/log": "^1.0 || ^2.0 || ^3.0"
     },
     "autoload": {
         "psr-4": {
             "Atlas\\Cli\\": "src/"
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "~7.0",
+        "phpunit/phpunit": "~7.0 || ~9.0",
```

Comment:
[Version 7 of PHPUnit is dead](https://phpunit.de/supported-versions.html), also [version 7 of PHP](https://www.php.net/supported-versions).